### PR TITLE
Add info tooltips next to properties

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-2734

Please double-check the Relayer description, currently ported one similar to LayerZero's explanation.

### CODE REVIEW

File: packages/frontend/src/view/components/protocol/RemoteChain.tsx [hunk 1]
```diff
@@ -67,40 +72,66 @@ export function RemoteChainComponent({
...
+              }
               value={remoteChain.defaultAppConfig.outboundProofType}
             />
             <Row
               hideBorder
               dense
               className="md:pl-7"
-              label="Oracle"
+              label={
+                <InfoTooltip text="A contract address that can be notified to move a block header. It writes a commitment for the messages on the destination chain.">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/protocol/RemoteChain.tsx:131]:
```suggestion
                <InfoTooltip text="An address allowed to advance a block header. The oracle moves a requested block header from a source chain to a destination chain. It writes a commitment for the messages on the destination chain.">
```

File: packages/frontend/src/view/components/protocol/RemoteChain.tsx [hunk 2]
```diff
@@ -111,7 +142,11 @@ export function RemoteChainComponent({
               hideBorder
               dense
               className="md:pl-7"
-              label="Relayer"
+              label={
+                <InfoTooltip text="Works with tandem with an Oracle to transmit messages between chains. By default, User Applications will use LayerZero Relayer which can be changed within the User Application's configuration.">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/protocol/RemoteChain.tsx:146]:
```suggestion
                <InfoTooltip text="Works in tandem with an Oracle to transmit messages between chains. Relayer proves the message inclusion in the source chain to the destination chain.">
```

File: packages/frontend/src/view/components/protocol/RemoteChain.tsx [hunk 3]
```diff
@@ -120,7 +155,13 @@ export function RemoteChainComponent({
             />
           </Block>
 
-          <Block title="Default adapter params">
+          <Block
+            title={
+              <InfoTooltip text="Adapter Parameters are a bytes-array used for specifying the amount of gas on the destination chain to send for an application to use. If this parameter is left blank or passed as 0x0, then defaults are inherited and used. The owner of UltraLightNodeV2 can set new default parameters.">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/protocol/RemoteChain.tsx:160]:
```suggestion
              <InfoTooltip text="Adapter Parameters are a bytes-array used for specifying the amount of gas on the destination chain to send for an application to use.">
```

File: packages/frontend/src/view/components/protocol/UltraLightNode.tsx [hunk 4]
```diff
@@ -32,7 +33,11 @@ export function UltraLightNodeContract(props: Props): JSX.Element {
       >
         <Subsection>
           <Row
-            label="Owner"
+            label={
+              <InfoTooltip text="The owner of the contract. Allowed to change the following values: the destination UltraLightNodeV2 library, the enabled outbound proof types, the available proof libraries, the default adapter parameters, and the default app config.">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/protocol/UltraLightNode.tsx:37]:
```suggestion
              <InfoTooltip text="The owner of the contract. Allowed to change the following values: the destination UltraLightNodeV2 library, the enabled outbound proof types, the available proof libraries, the default adapter parameters, the default app config, the Treasury contract address, and the LayerZero token. Allowed to withdraw fees from the Treasury contract.">
```

**Review by michalsidzej — CHANGES_REQUESTED** [review]
Couple of updated descriptions.

**Comment by sdlyy** [comment]
@michalsidzej rslvd

**Pull request merged**